From: Michael Froman <mjfroman@mac.com>
Date: Mon, 24 Jun 2024 09:15:58 -0500
Subject: (tmp-cherry-pick) Revert "Temporary disable sharding on Fuchsia
 bots." (36ecffa129)

This reverts commit 5dbc4a45bc3f8b6eee99426fb7ebfb76c861e8c2.

Reason for revert: Sharding should now work on fuchsia + no new generated starlark files have been uploaded since.

Original change's description:
> Temporary disable sharding on Fuchsia bots.
>
> Change-Id: I248ef19317de9a93751641e8bf0cab80a42a35f7
> Bug: b/338087169
> Reviewed-on: https://webrtc-review.googlesource.com/c/src/+/349840
> Commit-Queue: Mirko Bonadei <mbonadei@webrtc.org>
> Reviewed-by: Mirko Bonadei <mbonadei@webrtc.org>
> Cr-Commit-Position: refs/heads/main@{#42230}

Bug: b/338087169
Change-Id: I25f29a64a4a2ca40afb0ac9fbbc2a821c3ca4551
Reviewed-on: https://webrtc-review.googlesource.com/c/src/+/350182
Bot-Commit: rubber-stamper@appspot.gserviceaccount.com <rubber-stamper@appspot.gserviceaccount.com>
Reviewed-by: Mirko Bonadei <mbonadei@webrtc.org>
Commit-Queue: Jeremy Leconte <jleconte@google.com>
Cr-Commit-Position: refs/heads/main@{#42282}
---
 infra/specs/client.webrtc.json    | 6 ++++--
 infra/specs/test_suites.pyl       | 6 ++----
 infra/specs/tryserver.webrtc.json | 6 ++++--
 3 files changed, 10 insertions(+), 8 deletions(-)

diff --git a/infra/specs/client.webrtc.json b/infra/specs/client.webrtc.json
index 4eacdeb112..db0d5d4d75 100644
--- a/infra/specs/client.webrtc.json
+++ b/infra/specs/client.webrtc.json
@@ -2009,7 +2009,8 @@
           "dimensions": {
             "cpu": "x86-64",
             "os": "Ubuntu-22.04"
-          }
+          },
+          "shards": 4
         },
         "test": "svc_tests",
         "test_id_prefix": "ninja://pc:svc_tests/"
@@ -2057,7 +2058,8 @@
           "dimensions": {
             "cpu": "x86-64",
             "os": "Ubuntu-22.04"
-          }
+          },
+          "shards": 4
         },
         "test": "video_engine_tests",
         "test_id_prefix": "ninja://:video_engine_tests/"
diff --git a/infra/specs/test_suites.pyl b/infra/specs/test_suites.pyl
index 49b3bc9347..ee2dbc8f96 100644
--- a/infra/specs/test_suites.pyl
+++ b/infra/specs/test_suites.pyl
@@ -119,15 +119,13 @@
       # 'rtc_unittests': {},
       'rtc_pc_unittests': {},
       'svc_tests': {
-        # TODO: b/338087169 - Re-enable sharding.
-        # 'mixins': ['shards-4'],
+        'mixins': ['shards-4'],
       },
       'system_wrappers_unittests': {},
       # TODO(bugs.webrtc.org/14712):  Enable once network issue is fixed.
       # 'peerconnection_unittests': {},
       'video_engine_tests': {
-        # TODO: b/338087169 - Re-enable sharding.
-        # 'mixins': ['shards-4'],
+        'mixins': ['shards-4'],
       },
       'voip_unittests': {},
       'webrtc_nonparallel_tests': {},
diff --git a/infra/specs/tryserver.webrtc.json b/infra/specs/tryserver.webrtc.json
index 98cd16b11e..e448c19dec 100644
--- a/infra/specs/tryserver.webrtc.json
+++ b/infra/specs/tryserver.webrtc.json
@@ -2207,7 +2207,8 @@
           "dimensions": {
             "cpu": "x86-64",
             "os": "Ubuntu-22.04"
-          }
+          },
+          "shards": 4
         },
         "test": "svc_tests",
         "test_id_prefix": "ninja://pc:svc_tests/"
@@ -2255,7 +2256,8 @@
           "dimensions": {
             "cpu": "x86-64",
             "os": "Ubuntu-22.04"
-          }
+          },
+          "shards": 4
         },
         "test": "video_engine_tests",
         "test_id_prefix": "ninja://:video_engine_tests/"
